documentation/book/proc-downgrading-brokers-older-kafka.adoc

```diff
@@ -0,0 +1,87 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrading-kafka-versions.adoc
+
+
+[id='proc-downgrading-brokers-older-kafka-{context}']
+
+= Downgrading brokers to an older Kafka version
+
+You can downgrade a {ProductName} Kafka cluster from one version to a lower version; for example, from 2.1.0 to 2.0.0.
+
+IMPORTANT: Downgrading is not possible if the version being downgraded _from_ has ever used a `log.message.format.version` that's not supported by the version being downgraded _to_ (including where the default value for `log.message.format.version` is used). 
++For example, consider:
++
+[source,yaml]
+----
+apiVersion: v1alpha1
+kind: Kafka
+spec:
+  # ...
+  kafka:
+    version: 2.1.0
+    config:
+      log.message.format.version: "2.0"
+      # ...
+----
++
+This resource can be downgraded to Kafka version 2.0.0 because the `log.message.format.version` has not been changed.
+If the `log.message.format.version` were absent (so that the parameter took its default value for a 2.1.0 broker of 2.1), or was `"2.1"` then downgrade would not be possible.
+
+
+.Prerequisites
+
+* The Cluster Operator is up and running.
+* A `Kafka` resource to be downgraded.
+* The `Kafka.spec.kafka.config` has a `log.message.format.version` that is supported by the version being downgraded to.
+* You have checked that your `Kafka.spec.kafka.config` contains no options which are not supported in the version of Kafka being downgraded to.
+
+.Procedure
+
+. Change the `Kafka.spec.kafka.version` to specify the version being downgraded _to_.
+If the image to be used is different from the image for the given version of Kafka configured in the Cluster Operator's `STRIMZI_KAFKA_IMAGES` then configure the `Kafka.spec.kafka.image` as well.
++
+For example, if downgrading from Kafka 2.1.0 to 2.0.0:
++
+[source,yaml]
+----
+apiVersion: v1alpha1
+kind: Kafka
+spec:
+  # ...
+  kafka:
+    version: 2.0.0 <1>
+    config:
+      log.message.format.version: "2.0" <2>
+      # ...
+----
+<1> This is changed to the downgraded version
+<2> This is unchanged
++
+NOTE: It is necessary to format the value of `log.message.format.version` as a string to prevent it being interpreted as a number.
+
```

Comment:
```suggestion
+
```

Reply:
AFAICT this doesn't do anything. At least that's what the preview in vscode seems to show. Is this necessary?

Reply:
And in any case I thought we used + as a connector to make clear that a bunch of what would otherwise look like paragraphs following a list item were instead paragraphs in a list item. Since it's being used here to join one list item with the next clearly my understanding is wrong. Can you explain?

Reply:
I might be wrong, but since I don't have a rendered doc to look at, I'm guessing that because the 'NOTE' is not a bullet and needed a + to render in the right place, the numbered bullet will need the + as well so that it continues the numbering instead of restarting at 1.

Reply:
A `+` to connect the `NOTE` to the previous item is necessary. A `+` after the note seems to have no effect. Personally I think it's easier to read without this optional `+`, that way each item it separated by a blank line consistently like the rest of the docs.

Reply:
Okey dokey! As long as the numbers are in order and not restarting I'm a-ok.